base/src/main/java/org/aya/concrete/desugar/Desugarer.java

```diff
@@ -37,14 +37,14 @@ public static class DesugarInterruption extends Exception {}
 
   @Override public @NotNull Expr pre(@NotNull Expr expr) {
     return switch (expr) {
-      case Expr.AppExpr(var pos, Expr.RawSortExpr(var uPos, var kind), var arg)when kind == SortKind.Type -> {
+      case Expr.AppExpr(var pos,Expr.RawSortExpr(var uPos,var kind),var arg)when kind == SortKind.Type -> {
```

Comment:
Please upgrade IntelliJ

Reply:
I'm on 2022.2.3 now, but it seems that the formatting is still wrong.
Which version should I use then?

Reply:
EAP

Reply:
Ok, I've switched to EAP and some spaces are added back.
But there is still no space between `)` and `when`, is this fine at this point?

Reply:
Yes